imap_processing/cdf/config/imap_constant_attrs.yaml

```diff
@@ -0,0 +1,33 @@
+# By default the time is assumed to correspond to instantaneous time or
+# center of accumulation/measurement period. But if, for some good reason,
+# time corresponds to the beginning or end or any other part of
+# accumulation/measurement period, that has to be stated in CATDESC.
+epoch:
+  CATDESC: Time, number of nanoseconds since J2000 with leap seconds included
+  FIELDNAM: epoch
+  LABLAXIS: epoch
+  FILLVAL: -9223372036854775808
+  FORMAT: " " # Supposedly not required, fails in xarray_to_cdf
+  VALIDMIN: -9223372036854775808
+  VALIDMAX: 9223372036854775807
+  UNITS: ns
+  VAR_TYPE: support_data
+  SCALETYP: linear
+  MONOTON: INCREASE
+  TIME_BASE: J2000
+  TIME_SCALE: Terrestrial Time
+  REFERENCE_POSITION: Rotating Earth Geoid
+
+# <=== Data Variables ===>
+# Default Attrs for all metadata variables unless overridden
+metadata_attrs:
```

Comment:
This may go away when I switch function that was using `common_cdf_attrs.py` to new function `packet_file_to_datasets()`